www/class/centreonACL.class.php

```diff
@@ -2058,24 +2064,21 @@ public function getServiceServiceGroupAclConf($sg_id, $broker = null, $options =
                 . "AND $db_name_acl.centreon_acl.host_id = host.host_id "
                 . "AND $db_name_acl.centreon_acl.service_id = service.service_id ";
         }
+        /*
+         * Using the centreon_storage database to get the information
+         * where the services_servicegroups table provides "resolved" dependencies
+         * for possible components of the servicegroup which can be:
+         *     - simple services
+         *     - service templates
+         *     - hostgroup services
+         */
         $query = $request['select'] . $request['simpleFields'] . " "
             . "FROM ( "
             . "SELECT " . $request['fields'] . " "
-            . "FROM servicegroup, servicegroup_relation, service, host " . $from_acl . " "
-            . "WHERE servicegroup.sg_id = '" . CentreonDB::escape($sg_id) . "' "
-            . "AND service.service_activate='1' AND host.host_activate='1' "
```

Comment:
are you sure that you want results of disabled resources ?

Reply:
`AND service.service_activate='1' AND host.host_activate='1'` is this condition not enough to only get enabled ressources ?

If your question is "Do I want the disabled ressources ?" The answer is no :smiley:

Reply:
so you should restore this line ;)

Reply:
Should be good now :)